test/test_json_util.py

```diff
@@ -241,6 +243,69 @@ def test_datetime(self):
             ),
         )
 
+    def test_datetime_ms(self):
+        # Test ISO8601 in-range
+        dat_min = {"x": DatetimeMS(0)}
+        dat_max = {"x": DatetimeMS(_max_datetime_ms())}
+        opts = JSONOptions(datetime_representation=DatetimeRepresentation.ISO8601)
+
+        self.assertEqual(
+            dat_min["x"].to_datetime(tz_aware=False),
+            json_util.loads(json_util.dumps(dat_min))["x"],
+        )
+        self.assertEqual(
+            dat_max["x"].to_datetime(tz_aware=False),
+            json_util.loads(json_util.dumps(dat_max))["x"],
+        )
+
+        # Test ISO8601 out-of-range
+        dat_min = {"x": DatetimeMS(-1)}
+        dat_max = {"x": DatetimeMS(_max_datetime_ms() + 1)}
+
+        self.assertEqual("""{"x": {"$date": {"$numberLong": -1}}}""", json_util.dumps(dat_min))
```

Comment:
Here and below you should use single quotes instead of `"""`, for example `'{"x": {"$date": {"$numberLong": -1}}}'`